docs/src/main/asciidoc/kotlin.adoc

```diff
@@ -195,4 +392,4 @@ One thing to note is that the live reload feature is not available when making c
 
 == Packaging the application
 
-As usual, the application can be packaged using `./mvnw clean package` and executed using the `-runner.jar` file. You can also build the native executable using `./mvnw package -Pnative`.
\ No newline at end of file
+As usual, the application can be packaged using `./mvnw clean package` and executed using the `-runner.jar` file. You can also build the native executable using `./mvnw package -Pnative`, or `./gradlew buildNative` for the platform-local binary.  
```

Comment:
I don't think `for the platform-local binary` adds much value

Reply:
Removed.